src/sources/nats.rs

```diff
@@ -45,13 +45,20 @@ pub struct NatsSourceConfig {
     /// The NATS URL to connect to.
     ///
     /// The URL must take the form of `nats://server:port`.
+    #[configurable(metadata(docs::examples = "nats://demo.nats.io"))]
```

Comment:
Does this need a `:port` or does it assume `80` or something similarly default?

Reply:
Specifically the doc comment above this line indicates that the port must be included. If there is a default I think updating the comment to mention that would be a good idea.

Reply:
It turns out that it doesn't and defaults to 4222 if not specified. I've updated the docs.